tests/config/test_config_comparator.py

```diff
@@ -54,41 +57,119 @@ def test_config_comparator():
     _config_2._unique_sections[UniqueSectionForTest.name] = unique_section_1
 
     config_diff = _ConfigComparator(_config_1, _config_2)
+    assert config_diff.get("unblocked_diff_sections") is None
```

Comment:
Can we add a case where a non-blocking diff is detected?

Reply:
It is not on this test. There is one in the next test case. I'll need to rewrite the tests too.